packages/fxa-auth-server/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.136.2
+
+### Bug fixes
```

Comment:
Seems a bit strange this shows up twice, expected?

Reply:
Since we have some more patches to uplift, I'll keep this PR open an fix in the upcoming ones.